pyproject.toml

```diff
@@ -19,6 +19,7 @@ classifiers = ["Topic :: Scientific/Engineering :: Artificial Intelligence"]
 
 [tool.poetry.dependencies]
 python = "^3.10"
+typeguard = "<3.0.0"
```

Comment:
This will restrict all users of SAELens from installing any dependency that uses a modern version of typeguard, including all recent versions of tranformerlens. Why was this added?